### PR TITLE
New 8 model language and new 9 model category Voitovych

### PR DESCRIPTION
**Pull request by alexandrv1111**
created language and category models

### CODE REVIEW

**Review by MetalMari — APPROVED** [review]

**Comment by okslisovska** [comment]
Here I can see the following issues:
1.As we can see from migrations/0001_initial.py, the category id will be BigAutoField instead of ObjectIdField (the same for the language). The application throws an error about it (I described it in details in Slack channel project_process under my comment about this pull request). So to avoid this we need to add _id=...ObjectId... to the category model. Or (not 100% sure, but just noticed it)  maybe in  category/apps.py class CategoryConfig set somehow the default_auto_field to ObjectId?
2.Since we have 'ENFORCE_SCHEMA': False in settings.py DATABASES, I believe that we have no need to run migrations with MongoDB.

**Comment by okslisovska** [comment]
And if we are going to use Django's admin, we need our models to be registered at admin.py. But unfortunately, the admin still does not want to behave when I am testing( So I quit trying and quickly created a view function for testing. Anyway we are going to create our own admin for the site.

**Review by dmytrosmetanin — APPROVED** [review]
Approved

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,8 @@ jspm_packages/
 .env.development.local
 .env.test.local
 .env.production.local
+v
```
Comment (MrSancho42) [inline .gitignore]: Is this mistake?

Comment (vvelizariy) [inline .gitignore]: what is `v` stands for?

File: codeArena/category/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.6 on 2021-08-13 23:31
```
Comment (vvelizariy) [inline codeArena/category/migrations/0001_initial.py]: Can be omitted. Useless information.

File: codeArena/language/models.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# from django.db import models
+from djongo import models
+
+# Create your models here.
+class Language(models.Model):
+    title = models.CharField(max_length=100)
```
Comment (MrSancho42) [inline codeArena/language/models.py]: We named this field "name" in SQL Designer

File: codeArena/category/models.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# from django.db import models as m
+from djongo import models
+
+# Create your models here.
+
+class Category(models.Model):
+    title = models.CharField(max_length=100)
```
Comment (MrSancho42) [inline codeArena/category/models.py]: We named this field "name" in SQL Designer

File: codeArena/language/models.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+    def __str__(self):
+        return self.title
```
Comment (MrSancho42) [inline codeArena/language/models.py]: In end of file recommended to leave a empty line

**Review by MrSancho42 — CHANGES_REQUESTED** [review]

**Review by MrSancho42 — APPROVED** [review]

File: codeArena/category/models.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# from django.db import models as m
+from djongo import models
+
+# Create your models here.
+
```
Comment (MrSancho42) [inline codeArena/category/models.py:5]: This strings need to be removed

File: codeArena/category/models.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# from django.db import models as m
```
Comment (MrSancho42) [inline codeArena/category/models.py:1]: This string need to be removed

File: codeArena/language/models.py [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# from django.db import models
+from djongo import models
+
+# Create your models here.
```
Comment (MrSancho42) [inline codeArena/language/models.py:4]: Comments need to be removed

**Review by MrSancho42 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**